meshnet/exitnode/fw.go

```diff
@@ -211,7 +168,11 @@ type TrafficPeer struct {
 	LocalNetwork bool
 }
 
-func resetPeersTraffic(peers []TrafficPeer, commandFunc runCommandFunc) error {
+func resetPeersTraffic(peers []TrafficPeer, interfaceNames []string, commandFunc runCommandFunc) error {
```

Comment:
There will be merge conflicts with release branch. I think it's better to allow to merge release branch into main first and only then merge this PR.